bats/sql-server.bats

```diff
@@ -65,6 +65,57 @@ teardown() {
     [[ "$output" =~ "one_pk" ]] || false
 }
 
+
+@test "test dolt sql interface works properly with autocommit" {
+    skiponwindows "Has dependencies that are missing on the Jenkins Windows installation."
+
+    cd repo1
+    start_sql_server repo1
+
+    # No tables at the start
+    run dolt ls
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "No tables in working set" ]] || false
+
+    # create table with autocommit off and verify there are still no tables
+    server_query 0 "CREATE TABLE one_pk (
+        pk BIGINT NOT NULL COMMENT 'tag:0',
+        c1 BIGINT COMMENT 'tag:1',
+        c2 BIGINT COMMENT 'tag:2',
+        PRIMARY KEY (pk)
+    )" ""
+    run dolt ls
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "No tables in working set" ]] || false
+
+    # check that dolt_commit throws an error when autocommit is off
+    run dolt sql -q "SELECT DOLT_COMMIT('-a', '-m', 'Commit1')"
```

Comment:
This is wrong... any single command line SQL query like this should be auto committed by default.

Easiest way to fix this is probably to set the session variable when we start the command line SQL session.